app/views/browse/build-config.html

```diff
@@ -177,7 +177,6 @@
                             </td>
                             <td data-title="Created">
                               <relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp>
-                              <span>- {{build.metadata.creationTimestamp | date : 'short'}}</span>
```

Comment:
I believe @jwforres requested that the timestamp be removed from that column of the table, and to only show how long ago it was created.